src/System.Linq.Expressions/tests/Unary/UnaryConvertTests.cs

```diff
@@ -102,6 +102,18 @@ public static void ConvertArrayToIncompatibleTypeTest(bool useInterpreter)
             }
         }
 
+        [Fact]
+        public static void ToStringTest()
+        {
+            // NB: Unlike TypeAs, the output does not include the type we're converting to
+
+            var e1 = Expression.Convert(Expression.Parameter(typeof(object), "o"), typeof(int));
+            Assert.Equal("Convert(o)", e1.ToString());
```

Comment:
Interesting quirk: `TypeAs` includes the type we're converting to, but `Convert` and `ConvertChecked` don't.